casts/rows/src/UITableRow/UITableRow.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { TouchableOpacity } from '@tonlabs/uikit.controls';
+import { UICurrency, UINumber } from '@tonlabs/uicast.numbers';
+import { UILabel, UILabelColors, UILabelRoles } from '@tonlabs/uikit.themes';
+import { UILinkButton } from '@tonlabs/uikit.controls';
+import { UILayoutConstant, UISkeleton } from '@tonlabs/uikit.layout';
+
+import { UITableRowValueKind } from './types';
+import type { UITableRowValue, UITableRowProps } from './types';
+
+function renderTableValue(value: UITableRowValue) {
+    const { testID, kind, props } = value;
+
+    switch (kind) {
```

Comment:
I'm not sure that I like nested `kind`s, what about make it "plain"? Also do we really need it in UIKit, maybe better to keep it in Surf for particular place only, to not have arbitrary `props`?

Reply:
AFAIK, the plain rows look very differently from the design of "Table" related rows...